package.json

```diff
@@ -18,12 +18,12 @@
   },
   "devDependencies": {
     "babel-eslint": "^10",
-    "eslint": "^6 || ^7",
```

Comment:
It doesn't look like we need this in our `devDependencies` since we already have that in our `peerDependencies`.